app/lib/synthetic_branding_group_populator.rb

```diff
@@ -0,0 +1,107 @@
+##
+# SyntheticBrandingGroupPopulator: class to populate branding groups using the output of SyntheticStudyPopulator
+##
+
+class SyntheticBrandingGroupPopulator
+  BASE_BRANDING_GROUPS_CONFIG_PATH = Rails.root.join('db', 'seed', 'synthetic_branding_groups')
+
+  # populate all branding groups using synthetic studies
+  #
+  # @param user (User)                 => User account to own the branding groups
+  # @param overwrite_groups (Boolean)  => T/F for destroying/recreating any existing groups
+  # @param overwrite_studies (Boolean) => T/F for destroying/recreating any existing studies
```

Comment:
When I read these comments, I thought that if overwrite was false, the groups wouldn't be destroyed&recreated, but would still be updated-in-place.  I think it's relatively uncommon to run a populator and *not* have it update existing elements (remember when we kept getting screwed by SearchFacetPopulator not updating existing facets?).  So if you want to keep this behavior, I would suggest renaming the option to update_existing (and then inverting the logic) so it's really clear what you mean

Reply:
568c460, 7290786